lib/core/src/display/symbol.rs

```diff
@@ -25,6 +25,7 @@ use crate::system::gpu::shader::Context;
 use crate::system::gpu::data::buffer::IsBuffer;
 use crate::system::gpu::data::uniform::AnyUniform;
 use crate::system::gpu::data::uniform::AnyTextureUniform;
+use crate::system::gpu::data::uniform::AnyTextureUniformOps;
 use crate::system::gpu::data::uniform::AnyPrimUniform;
 use crate::system::gpu::data::uniform::AnyPrimUniformOps;
```

Comment:
I'll fix it, but I admit that for me it not convenient form: 
a) as a code writer: before I could use some structs not knowing where they are, because IDE find them for me. Now I must find, in which _types_ crate they are
b) as a code reader: before I could see in file without IDE support from where the struct/trait was used. Now I don't know from which `types::*` the object comes.

And usually if I write code, I do it in IDE, but often read code somewhere else (e.g. doing a review).